README.md

```diff
@@ -138,7 +138,8 @@ If they do not match or one is missing, please [reinstall the package](README.md
     jupyter labextension install @jupyterlab/git
     ```
     
-    If you see `@jupyterlab/git` under the  `Uninstalled core extensions: ` remove jupyter\lab\settings\build_config.json and re-install.
+    If you see `@jupyterlab/git`, your installation may have been corrupted. You can run `jupyter lab clean --all` and
```

Comment:
It looks like the first part of the sentence is missing some words now? 

"If you see `@jupyterlab/git` in ____" perhaps?

Reply:
Here is the output of the corrupted setup, @jupyterlab/git  is installed but also reported under the `Uninstalled core extensions: `, the sentence can be phrased as  "If you see @jupyterlab/git in Uninstalled core extensions ..."  
JupyterLab v3.0.9
/anaconda3/envs/jupyter38/share/jupyter/labextensions
        @jupyterlab/git v0.30.0-beta.2 enabled OK (python, jupyterlab-git)
        @jupyter-widgets/jupyterlab-manager v3.0.0 enabled OK (python, jupyterlab_widgets)

Uninstalled core extensions:
    @jupyterlab/git

Reply:
The idea was to get the negation of 135. But definitely it is not clear enough. Should we restrain it to the _uninstalled_ case of @EnisBerk or use `jupyter lab clean --all` as a fallback in abnormal case.